tasks/replace_package_json.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function (grunt) {
+  grunt.registerTask('replace_package_json', function () {
+    var path = grunt.config.process('<%= build %>/kibana/package.json');
+    var pkg = grunt.config.get('pkg');
+
+    grunt.file.write(path, JSON.stringify({
+      name: pkg.name,
+      description: pkg.description,
+      keywords: pkg.keywords,
+      version: pkg.version,
+      buildNum: grunt.config.get('buildNum'),
```

Comment:
As pedantic as this is, I really dislike the camel casing here. What about tossing this under a `build` property, like `build: { num: ... , commit: ... }`. This would also allow us to include other metadata easily, like the build date, the user (or server) that built it and whatever else we might deem interesting to append.